config/static_data.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+# class with some constants and static data
+class StaticData
```

Comment:
I don't think this should be in separate StaticData class

It can be placed in `logger_helper.rb` class safely